### ISSUE TITLE
GH marketplace still shows v1.2.1 as latest version

### ISSUE THREAD

**Issue by powerman** [golangci/golangci-lint-action#87]
Not sure is you can do anything about this or it should be reported somewhere at GH marketplace, opened issue here just to let you know.

**Comment by SVilgelm** [comment]
@jirfag Could please teak a look at this?

**Comment by MaxPeal** [comment]
ping @golangci/core-team @jirfag

**Comment by coufalja** [comment]
This is becoming an issue if you or your organisation has set the security policy
```
golangci/golangci-lint-action@v2.3.0 is not allowed to be used in xxx. Actions in this workflow must be: created by GitHub, verified in the GitHub Marketplace, within a repository owned by wandera or match the following: xxx
```

**Comment by SVilgelm** [comment]
@golangci/core-team please check it

**Comment by SVilgelm** [comment]
@golangci/core-team 
@ernado @jirfag 
Guys, could you please check the settings and accept an agreement? We cannot publish the new versions to the marketplace.
<img width="724" alt="Screen Shot 2021-02-14 at 11 55 06 AM" src="https://user-images.githubusercontent.com/523825/107884658-c2bc6b80-6ebb-11eb-9b88-b59d05c6e96c.png">

**Comment by ernado** [comment]
> golangci must accept the GitHub Marketplace Developer Agreement. Contact an org admin to accept it.

Can't do it myself, will pm @jirfag soon

**Comment by SVilgelm** [comment]
@jirfag thank you! The issue is fixed

**Comment by jizi** [comment]
Unfortunately `golangci` is not "verified creator" so with organization policy set we cannot use it from marketplace even with this issue resolved. Anyway we can use it from our fork.

**Comment by SVilgelm** [comment]
@jirfag Is it possible to verify the org? 
https://docs.github.com/en/developers/github-marketplace/about-marketplace-badges

**Comment by ldez** [comment]
https://github.com/golangci/golangci-lint-action/issues/579#issuecomment-1312335981
